### PR TITLE
add change log entries

### PR DESCRIPTION
**Pull request by taharallouche**
## Description
Hello @TamaraAtanasoska here are the entries that I think are relevant. I didn't include the changes related to refactoring, type hints, and testing (namely the mitigation statistical testing PR) as they are not user-facing.

## Tests

- [x] no new tests required
- [ ] new tests added
- [ ] existing tests adjusted

## Documentation

- [ ] no documentation changes needed
- [x] user guide added or updated
- [ ] API docs added or updated
- [ ] example notebook added or updated

### CODE REVIEW

**Comment by TamaraAtanasoska** [comment]
@taharallouche thanks, all set for the release from this point! there are some CI issues I am working on to fix (external failures), I will merge this after I merge those fixes as CI won't pass otherwise.

**Comment by taharallouche** [comment]
@TamaraAtanasoska Sorry I just noticed your comment about the new format in the other PR :eyes: I'll try to fix it tomorrow morning

**Comment by TamaraAtanasoska** [comment]
@taharallouche no worries I just updated the format and snuck a line I forgot from mine 😄 . thanks so much for all the work you did this release! After CI passes I will merge it and start the release tomorrow.

**Review by TamaraAtanasoska — APPROVED** [review]

**Pull request merged**
